meta-cube/recipes-support/overc-utils/source/cube-cfg

```diff
@@ -550,6 +550,7 @@ IFS='
 	             ${tty_flag} \
 	             ${hooks} \
 		     ${app_args} \
+                     --linux-cgroups-path=/runc/${container_name} \
```

Comment:
We should put this in /var/lib/cube/ with the rest of the volatile bits of the containers. Otherwise we are starting to spray around the FS (like docker).

Reply:
ok. never mind this one. my misunderstanding.